test/test_plugin_workflows.py

```diff
@@ -0,0 +1,433 @@
+    }),
+    ('workflows://host:443/workflow1e/signature/?api-version=2024-01-01', {
+        # support api-version which is extracted from webhook
+        'instance': NotifyWorkflows,
+        # Our expected url(privacy=True) startswith() response
+        'privacy_url': 'workflow://host:443/w...e/s...e/',
+    }),
+    ('workflows://host:443/workflow1b/signature/?ver=2016-06-01', {
+        # Support ver= (api-version alias)
+        'instance': NotifyWorkflows,
+    }),
+    ('workflows://host:443/?id=workflow1b&signature=signature', {
+        # Support signature= (sig= alias)
+        'instance': NotifyWorkflows,
+        # Our expected url(privacy=True) startswith() response
+        'privacy_url': 'workflow://host:443/w...b/s...e/',
+    }),
+    # Support native URLs
+    ('https://server.azure.com:443/workflows/643e69f83c8944/'
+     'triggers/manual/paths/invoke?'
+     'api-version=2016-06-01&sp=%2Ftriggers%2Fmanual%2Frun&'
+     'sv=1.0&sig=KODuebWbDGYFr0z0eu', {
+         # All tokens provided - we're good
+         'instance': NotifyWorkflows,
+
+         # Our expected url(privacy=True) startswith() response
+         'privacy_url': 'workflow://server.azure.com:443/6...4/K...u/'}),
+
+    ('workflow://host:443/workflow2/signature/', {
+        'instance': NotifyWorkflows,
+        # force a failure
+        'response': False,
+        'requests_response_code': requests.codes.internal_server_error,
+    }),
+    ('workflow://host:443/workflow3/signature/', {
+        'instance': NotifyWorkflows,
+        # throw a bizzare code forcing us to fail to look it up
+        'response': False,
+        'requests_response_code': 999,
+    }),
+    ('workflow://host:443/workflow4/signature/', {
+        'instance': NotifyWorkflows,
+        # Throws a series of connection and transfer exceptions when this flag
+        # is set and tests that we gracfully handle them
+        'test_requests_exceptions': True,
+    }),
+)
+
+
+def test_plugin_workflows_urls():
+    """
+    NotifyWorkflows() Apprise URLs
+
+    """
+
+    # Run our general tests
+    AppriseURLTester(tests=apprise_url_tests).run_all()
+
+
+@pytest.fixture
+def workflows_url():
+    return 'workflow://host:443/workflow/signature'
+
+
+@pytest.fixture
+def request_mock(mocker):
+    """
+    Prepare requests mock.
+    """
+    mock_post = mocker.patch("requests.post")
+    mock_post.return_value = requests.Request()
+    mock_post.return_value.status_code = requests.codes.ok
+    return mock_post
+
+
+@pytest.fixture
+def simple_template(tmpdir):
+    template = tmpdir.join("simple.json")
+    template.write(cleandoc("""
+    {
+        "type": "message",
+        "attachments": [{
+            "contentType": "application/vnd.microsoft.card.adaptive",
+            "contentUrl": None,
+            "content": {
+                "$schema":"http://adaptivecards.io/schemas/adaptive-card.json",
+                "type": "AdaptiveCard",
+                "version": "1.4",
```

Comment:
Could we add the following to the adaptive card, so that for long messages it will occupy the full width of the Teams window?
```suggestion
                "version": "1.4",
                "msteams": { "width": "full" },
```

Reply:
Of course!